src/main/kotlin/com/avail/builder/ModuleNameResolver.kt

```diff
@@ -41,8 +41,8 @@ import java.util.*
 
 /**
  * A `ModuleNameResolver` resolves fully-qualified references to Avail
- * [modules][ModuleDescriptor] to [absolute][File.isAbsolute] [file
- * references][File].
+ * [modules][ModuleDescriptor] to [absolute][File.isAbsolute]
+ * [file references][File].
```

Comment:
&#32;

Reply:
It posted your comment as nothing at all — no body — but I see what you want fixed, and it's fixed.

Reply:
FYI. When I edit your comment to see the source, it shows that you typed `&#32;`, but when you don't use code quotes it just posts a space.